forecasting/src/autogluon/forecasting/utils/utils/savers/save_pickle.py

```diff
@@ -0,0 +1,12 @@
+import pickle
+import os
+
+
+def save(path, obj):
```

Comment:
Can you use the existing pickle-save function instead?

https://github.com/awslabs/autogluon/blob/master/core/src/autogluon/core/utils/savers/save_pkl.py#L10

Reply:
I am already using the existing pickle functions. These files are no longer in use and I will delete them in my next update to this PR.